scene/resources/curve.h

```diff
@@ -309,7 +313,8 @@ class Curve3D : public Resource {
 	Vector3 get_closest_point(const Vector3 &p_to_point) const;
 	real_t get_closest_offset(const Vector3 &p_to_point) const;
 
-	PackedVector3Array tessellate(int p_max_stages = 5, real_t p_tolerance = 4) const; //useful for display
+	PackedVector3Array tessellate(int p_max_stages = 5, real_t p_tolerance = 4) const; // Useful for display.
+	PackedVector3Array tessellate_even_length(int p_max_stages = 5, real_t p_length = 0.2) const; // Useful for baking.
```

Comment:
Is there any reason to keep the old original tessellate?

Reply:
Yes. For the drawing code the old method is more efficient as it doesn't break long straight segments.

Reply:
Okay, thank you for explanation.